db/migrate/20200330142218_add_at_home_to_motifs.rb

```diff
@@ -0,0 +1,5 @@
+class AddAtHomeToMotifs < ActiveRecord::Migration[6.0]
+  def change
+    add_column :motifs, :at_home, :boolean, default: false
```

Comment:
So I double checked and the `after: :existing_column_name` is a valid and native option for `add_column`. I think it's a good idea to start using it to keep coherent columns order in the db

Reply:
À quelle position on place cette colonne là ? À coté de `by_phone` ?

Reply:
yes, aside `by_phone` sounds good, but maybe let's discuss the schema options first? https://github.com/betagouv/rdv-solidarites.fr/pull/506#issuecomment-606115881